public-ui/src/app/views/ehr-view/ehr-view.component.html

```diff
@@ -1,245 +1,264 @@
+              *ngIf="synonymString[r.conceptId].length > 100"> See More
+              <clr-icon shape="ellipsis-horizontal"></clr-icon>
+            </a>
+          </ng-template>
         </div>
-        <div class="col-xs-12 row-expansion flex-xs-middle" *ngIf="r.expanded">
-          <ng-container *ngIf="!showMeasurementGenderBins">
-            <div class="col-md-4">
-              <button *ngFor="let g of ['Biological Sex','Gender Identity']" [ngClass]="{active:selectedGenderGraph === g}" (click)="selectGenderGraph(g)" class="btn btn-link">{{g}}
-              </button>
+        <div class="row-expansion" *ngIf="r.expanded" #chartElement>
+          <div class="graph-menu">
+            <div *ngFor="let g of ['Biological Sex', 'Gender Identity', 'Age at Occurrence', 'Sources']"
+              [ngClass]="{'active':graphToShow === g}" (click)="selectGraph(g)" class="chart-choice">
+              <span>{{g}}</span>
               <clr-tooltip>
-                <clr-icon clrTooltipTrigger shape="info-standard" class="info-icon"></clr-icon>
-                <clr-tooltip-content clrPosition=right
-                                 clrSize="lg" *clrIfOpen>
-                  <div class="tooltip-label">Toggle here to view biological sex / gender identity chart</div>
+                <clr-icon clrTooltipTrigger shape="info-standard" class="is-solid info-icon"></clr-icon>
+                <clr-tooltip-content clrPosition=top-right clrSize="lg" *clrIfOpen>
+                  <div class="tooltip-label">{{showToolTip(g)}}</div>
                 </clr-tooltip-content>
               </clr-tooltip>
             </div>
-          </ng-container>
-          <app-concept-charts [concept]="r" [showMeasurementGenderBins]="showMeasurementGenderBins" [showAge]="showAge" [showGender]="showGender" [showGenderIdentity]="showGenderIdentity"
-                            [showSources]="showSources" [backgroundColor]="'#ECF1F4'">
+          </div>
+          <app-concept-charts [concept]="r" [showGraph]="graphToShow" [backgroundColor]="'#ECF1F4'">
           </app-concept-charts>
         </div>
-        </div>
+
+
+
       </div>
+    </div>
 
-      <div class="results-grid">
-        <div class="row" *ngIf="!loading && items.length === 0">
-          <div class="col-xs-12">
-            <h5 class="no-results"> No results match your search. </h5>
-          </div>
+
+
+    <div class="results-grid">
+      <div *ngIf="!loading && items.length === 0">
+        <div class="">
+          <h5 class="no-results"> No results match your search. </h5>
         </div>
-        <div class="row">
-          <div class="col-md-4 col-xs-12">
-            <button class="btn btn-primary btn-sm" [routerLink]="['/quick-search/ehr']"> Back to Quick Search </button>
-          </div>
+      </div>
+      <div>
+        <div class="col-md-4 ">
+          <button class="btn btn-primary btn-sm" [routerLink]="['/quick-search/ehr']"> Back to Quick Search </button>
```

Comment:
I observed if I click any domain to see charts and then scroll all the way down and click back to quick search button , it goes back to quicksearch but I can see empty space , I need to scroll up to see the EHR Domains.
![screen shot 2019-02-27 at 3 44 06 pm](https://user-images.githubusercontent.com/42041705/53525484-7fd31480-3aa7-11e9-95bc-1a02ba706090.png)